src/libs/ReportActionsUtils.js

```diff
@@ -100,7 +100,7 @@ function hasCommentThread(reportAction) {
  * @param {Object} [allReportActionsParam]
  * @returns {Object}
  */
-function getParentReportAction(report, allReportActionsParam = {}) {
+function getParentReportAction(report, allReportActionsParam) {
```

Comment:
This param is optional, so it'd be better to do `allReportActionsParam = undefined`, like we do in a few other places, e.g. https://github.com/Expensify/App/blob/450573ebd01e710023940232da079928b41726a3/.github/libs/ActionUtils.js#L28

Reply:
Ahh good point

Reply:
I thought about that, but then I thought it would be equivalent, is it not?

Reply:
It is, it's just more explicit.